docs/howto/openstack/magnum/new-k8s-cluster.md

```diff
@@ -66,87 +78,129 @@ package manager of your operating system or `pip`:
-    | Name    | Fingerprint                                     | Type |
-    +---------+-------------------------------------------------+------+
-    | husavik | 34:3b:58:ba:ec:95:f5:17:17:df:04:38:11:89:e6:3d | ssh  |
-    +---------+-------------------------------------------------+------+
+    ```console
+    $ openstack keypair list
+    +----------+-------------------------------------------------+------+
+    | Name     | Fingerprint                                     | Type |
+    +----------+-------------------------------------------------+------+
+    | lefkanti | e7:e9:c5:95:ee:7b:72:37:3c:89:c5:fc:6e:8c:a1:72 | ssh  |
+    +----------+-------------------------------------------------+------+
     ```
 
     ...and set the `KEYPAIR` variable to the name of the keypair you
     want:
 
-    ```bash
-    KEYPAIR="husavik" # again, this is just an example
+    ```console
+    $ KEYPAIR="lefkanti" # this is just an example
     ```
 
+    Besides the OS disk, all cluster nodes have an extra disk for permanently storing application data.
+    The size of this extra disk is specified during cluster creation via the `--docker-volume-size` parameter.
+    Staying loyal to the theme of setting parameters via environment variables, decide on the size of this extra volume like so:
+
+    ```console
+    $ PERSIST_VOL_SIZE=50
+    ```
+
+    The size is expressed in Gibibytes, and in the example above, we decided to go with a 50GiB extra volume.
     Finally, decide on a name for your new Kubernetes cluster:
 
-    ```bash
-    CLUSTER_NAME="bangor"
+    ```console
+    $ CLUSTER_NAME="bangor"
     ```
 
     With everything in place, go ahead and create your new Kubernetes
-    cluster:
+    cluster like so:
 
-    ```bash
-    openstack coe cluster create \
-        --cluster-template $CLUSTER_TMPL \
-        --keypair husavik
-        bangor
+    ```console
+    $ openstack coe cluster create \
+        --cluster-template "$CLUSTER_TMPL" \
+        --keypair $KEYPAIR \
+        --docker-volume-size $PERSIST_VOL_SIZE \
+        $CLUSTER_NAME
     ```
 
-    If everything went well with your request for a new cluster, on your
-    terminal, you would see a message like the following:
+    New Magnum clusters start with 1 master node and 1 worker node by default.
+    Since the new cluster templates have the master load balancer enabled,
+    you might want to give your cluster 3 master nodes from the get-go.
+    Optionally, you can increase the number of worker nodes from 1 to 3 or to 5.
+    Contrary to the number of master nodes, you can always change the number of
+    worker nodes *after* the cluster is created. So, to start the new cluster
+    with 3 master nodes and 5 worker nodes, type the following:
+
+    ```console
+    $ openstack coe cluster create \
+        --cluster-template "$CLUSTER_TMPL" \
+        --keypair $KEYPAIR \
+        --docker-volume-size $PERSIST_VOL_SIZE \
+        --master-count 3 \
+        --node-count 5 \
```

Comment:
It's not *completely* logical to me why we want to set the persistent volume size with a variable, while just using int literals for the master and worker node count. Is there a specific reason to go with the variable, rather than just `50`?

Reply:
Well, we start with shell variables to show what a _basic_ cluster creation command looks like. A bit later during our demonstration, we make a somewhat parenthetical comment on the `--master-count` and `--node-count` parameters, and we give a quick but specific example of creating a cluster with `3` masters and `5` workers. We _specifically_ mention `3` and `5`, so introducing shell variables for _that_ particular example wouldn't make much sense, I believe.